curriculum/challenges/english/25-front-end-development/lab-fortune-teller/66c06d618d075c7f7f1b890a.md

```diff
@@ -0,0 +1,144 @@
+---
+id: 66c06d618d075c7f7f1b890a
+title: Build a Fortune Teller
+challengeType: 14
+dashedName: build-a-fortune-teller
+---
+
+# --description--
+
+In this lab, you will define five fortunes and randomly select one of them to display the fortune to the user.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
```

Comment:
```suggestion
**Objective:** Complete the following user stories and ensure all tests pass to complete the lab.
```

Reply:
We are using that sentence for all the labs @Sembauke Personally, I'd keep it as is.

Reply:
I would just keep it like that then too. We should use that for now on all the labs and we can go back and change them all at once if we want some else.